.github/scripts/auto-label.cjs

```diff
@@ -23,6 +23,10 @@ const prefixLabelMap = {
   '.eslintrc.js': 'style',
   '.github/workflows/': 'ci',
   '.github/scripts/': 'ci',
+  'plugin/': 'plugin',
+  'docs/': 'docs',
+  'screenshots/': 'docs',
+  'README.md': 'docs',
```

Comment:
This is a little bit suspicious, I know.  I could change all of these prefixes to be regexes instead, and then I could match .md files anywhere?  Maybe that would be better?

Reply:
I think `*.md` for just the top level might be a fair compromise here? Netting you all of README, CONTRIBUTING, and CODE_OF_CONDUCT? I think most other Markdown files I would want to fit this would be in `docs/`, and would also get labelled docs.
This wouldn't catch something like [.github/workflows/README.md](https://github.com/quisquous/cactbot/blob/main/.github/workflows/README.md); I can go either way on that, personally.

Reply:
Why not just add a file extension checking?

Reply:
There is a lot of regex in cactbot.  I think it'd be better to continue to use that then to add some other pattern matching mechanism.

Reply:
we have been using minimatch in lint-staged config

Reply:
Hmm, that's fair.